nncf/common/graph/transformations/commands.py

```diff
@@ -17,20 +17,47 @@
 
 
 class TransformationPriority(OrderedEnum):
+    """
+    Defines priorities for compression and service operations that are
+    added as modifications to the original model graph in order to turn it into a
+    'compressed' model, i.e. a model that supports compression-aware training and
+    export. Certain compression algorithms may need to act on one and the same
+    spot in the model's control flow graph, and proper priority-based ordering is
+    required for functionally correct results that are invariant w.r.t. compression
+    algorithm application ordering.
+
+    Rationales:
+    * quantization should occur after sparsification/pruning, otherwise the dependency
+    of sparsity level on the value threshold becomes discontinuous which impacts the
+    corresponding algos.
+    """
     DEFAULT_PRIORITY = 0
     FP32_TENSOR_STATISTICS_OBSERVATION = 1
     PRUNING_PRIORITY = 2
     SPARSIFICATION_PRIORITY = 3
     QUANTIZATION_PRIORITY = 11
 
 
-class TransformationType(OrderedEnum):
-    INSERT = 0
-    MULTI_INSERT = 1
-    REMOVE = 2
-
-
 class TargetType(OrderedEnum):
+    """
+    Describes the types of actions that can be applied to the NNCF internal graph representation
+    of the compressed model.
+
+    Definitions used below: TF-nodes - visible in TF-graph, TF-operations - invisible in TF-graph
```

Comment:
The description looks confusing to me. Some points are below: 
- TargetType is not a type of action, but It is an object or location where the action will be applied.
- What is TF-graph? It can be confused for developers because TF has several graphs DefGraph, KerasGraph and etc.
- What are visible and invisible nodes?
- What do you mean by hooking in TF?

Reply:
I modified the description to resolve some of your bullets here, but my knowledge of TF graph and node specifics is lacking in order to give a correct description of TF entities. Please suggest exact wording of the descriptions, containing exact TF entity identifiers (such as qualified class names), so that there is no ambiguity in the description. I tried to follow the existing code to try and obtain this knowledge on my own, but either the code is too convoluted, or it does not explicitly involve "operations" or "nodes".